lib/typescript/assets/scss/fonts.scss

```diff
@@ -2,56 +2,56 @@
 
 @mixin font-s {
   font-family: 'Lato', sans-serif;
-  font-size: 0.8rem; // 13px
-  line-height: 1rem;
+  font-size: 13px;
+  line-height: 16px;
 }
 
 @mixin font-base {
   font-family: 'Lato', sans-serif;
-  font-size: 1rem; // 16 px
-  line-height: 1.5rem;
+  font-size: 16px;
+  line-height: 24px;
 }
 
 @mixin font-m {
   font-family: 'Lato', sans-serif;
-  font-size: 1.25rem; // 20px
-  line-height: 2rem;
+  font-size: 20px;
+  line-height: 32px;
 }
 
 @mixin font-l {
   font-family: 'Lato', sans-serif;
-  font-size: 1.563rem; // 25px
-  line-height: 2rem;
+  font-size: 25px;
```

Comment:
@AitorAlgorta what's the reason for switching this to pixel's? If there is a good reason we should document it, because as far as I can understand `rem` scales while pixels are fixed size.

Reply:
That's why. `rem` not being a fix size brought some problems in different customer webpages. We want the chat plugin to look always the same no matter what and with `rem` it was looking really weird. Messages had una size. the header bar hada different one, the input a different one. I changed it to `px` so it always looks the same. @chrismatix

Reply:
![Screenshot 2021-12-08 at 10 22 46](https://user-images.githubusercontent.com/6797037/145182758-620e2f40-8094-4258-a814-4c35cacec221.png)

Reply:
Look at this ridiculous difference in sizes. We cannot control how `rem` behaves in different customer webpages, with `px` we have more control over it.

Reply:
Makes sense. Was not thinking about it in the context of the chatplugin 👍